docs/docs/step-by-step-walkthrough.md

```diff
@@ -0,0 +1,128 @@
+# Step by Step QHub Cloud Deployment
+
+1) **Environment variables**
+
+    Set the required environment variables based on your choice of provider below:
+
+- [AWS Environment Variables](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/aws/installation.md).
+- [Digital Ocean Environment Variables](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/do/installation.md)
+- [Google Cloud Platform](https://github.com/Quansight/qhub/blob/ft-docs/docs/docs/gcp/installation.md)
```

Comment:
I guess these links will be updated before merging, as they point to `ft-docs` branch at the moment.